tests/perf/linux_perf_event.cc

```diff
@@ -90,7 +90,9 @@ linux_perf_event::user_instructions_retired() {
             .disabled = 1,
             .exclude_kernel = 1,
             .exclude_hv = 1,
+#if !defined(__aarch64__)
             .exclude_idle = 1,
+#endif
```

Comment:
Where is it documented? Perhaps we should opt-in where it is supported?

Reply:
Also, need some comment, it looks completely obscure.

Reply:
https://man7.org/linux/man-pages/man2/perf_event_open.2.html

Reply:
```
       EPERM  Returned on many (but not all) architectures when an
              unsupported exclude_hv, exclude_idle, exclude_user, or
              exclude_kernel setting is specified.
```

Reply:
Since we've lived without it until https://github.com/scylladb/seastar/commit/93f19d39526c68defac6860a15c77160da273606 we can opt-in and include it only on x86_64 where we know it works (and add a comment about that)